packages/lib-react-components/src/Media/components/Video/Video.js

```diff
@@ -28,9 +28,7 @@ export default function Video({
       maxHeight={height}
       width='100%'
     >
-      <GrommetVideo a11yTitle={alt} controls={controlsOption} fit={fit} preload='metadata' src={src}>
```

Comment:
Note that I did not turn off `controls` for all Video components in lib-react-components. That's because Media's Video component is used if project teams want to display a video in markdown in the field guide, tutorial, or About Project pages. In those cases, we do want to display controls.